test/remote/gateways/remote_orbital_test.rb

```diff
@@ -40,6 +115,420 @@ def test_successful_purchase
     assert_equal 'Approved', response.message
   end
 
+  # def test_successful_purchase_with_soft_descriptor_hash
+  #   options = @options.merge(
+  #     soft_descriptors: {
+  #       merchant_name: 'Merch',
+  #       product_description: 'Description',
+  #       merchant_email: 'email@example'
+  #     }
+  #   )
+  #   assert response = @gateway.purchase(@amount, @credit_card, options)
+  #   assert_success response
+  #   assert_equal 'Approved', response.message
+  # end
+
+  def test_successful_purchase_with_card_indicators
+    options = @options.merge(
+      card_indicators: 'y'
+    )
+    assert response = @gateway.purchase(@amount, @credit_card, options)
+    assert_success response
+    assert_equal 'Approved', response.message
+  end
+
+  def test_successful_purchase_with_card_indicators_and_line_items
+    options = @options.merge(
+      line_items: @line_items,
+      card_indicators: 'y'
+    )
+    assert response = @gateway.purchase(@amount, @credit_card, options)
+    assert_success response
+    assert_equal 'Approved', response.message
+  end
+
+  def test_successful_purchase_with_level_2_data
```

Comment:
do we use level 2 data?

Reply:
We are not, I am gonna comment out this.